esp-hal/MIGRATING-0.23.md

```diff
@@ -23,3 +23,24 @@ Thus, explicit initialization of all fields can be replaced by only the necessar
       )
      .unwrap();
 ```
+
+## `Async` drivers can no longer be sent between tasks and cores
+
+To work around this limitation, send the blocking driver, and configure it into `Async` mode
+in the target context.
+
+```diff
+ #[embassy_executor::task]
+-async fn interrupt_driven_task(mut spi: Spi<'static, Async>) {
++async fn interrupt_driven_task(spi: Spi<'static, Blocking>) {
++    let mut spi = spi.into_async();
+     ...
+ }
+
+ let spi = Spi::new(...)
+     .unwrap()
+     // ...
+-    .into_async();
+
+ spawner.spawn(interrupt_driven_task(spi)).unwrap();
```

Comment:
maybe I'm missing something, but a regular spawner allows !Send - `make_send` make it loose the ability to spawn non-Send tasks